apps/projects/app/components/Content/Issues.js

```diff
@@ -159,14 +159,16 @@ class Issues extends React.PureComponent {
       // if there are no Status filters, all issues pass
       if (Object.keys(filters.statuses).length === 0) return true
       // should bountyless issues pass?
-
       const status = bountyIssueObj[issue.number] ? bountyIssueObj[issue.number] : 'not-funded'
       if ('not-funded' in filters.statuses && !bountyIssueObj[issue.number])
```

Comment:
This line is made redundant by `status in filters.statuses` as `if status in not-funded` `bountyIssueObj[issue.number]` is _necesarily_ false